packages/snaps-utils/src/npm.ts

```diff
@@ -78,11 +90,24 @@ export function validateNpmSnap(
     }
   }
 
+  if (localizationFiles) {
+    try {
+      getValidatedLocalizationFiles(localizationFiles);
```

Comment:
Perhaps leave a comment to explain why we are calling something that doesn't look like a validation function here?